app_dart/lib/src/request_handlers/update_task_status.dart

```diff
@@ -105,7 +105,12 @@ class UpdateTaskStatus extends ApiRequestHandler<UpdateTaskStatusResponse> {
       await _insertBigquery(commit, task);
     }
 
-    await _writeToMetricsCenter(resultData, scoreKeys, commit, task);
+    try {
+      await _writeToMetricsCenter(resultData, scoreKeys, commit, task);
+    } catch (e) {
```

Comment:
From the log, the error is `DetailedApiRequestError`, do you think we can use the exact error?